docs/content/v2024.2/deploy/checklist.md

```diff
@@ -174,6 +174,121 @@ YugabyteDB stores data compressed by default. The effectiveness of compression d
+
+- Frequent backups: Increase your backup frequency when using ephemeral storage. YBA allows for incremental backups to occur as frequently as every 15 minutes. Also, consider backing up to a diversity of backup storage, and performing regular test restores.
+
+#### Operational best practices and workflows
+
+Working with ephemeral storage requires careful attention to standard operational workflows. The following are best practices for common scenarios.
+
+##### Rolling restart of servers
+
+The [Rolling restart](../../yugabyte-platform/manage-deployments/edit-config-flags/#batched-rolling-restart) universe action in YBA does not reboot the machine or unmount disks, so ephemeral data remains intact. You can perform rolling restarts as usual.
+
+##### Operating System Patching and Node Reboots
+
+OS Patching procedures for self-managed clusters are customer-driven via scripts that call YBA APIs. See [Patch and upgrade the Linux operating system](../../yugabyte-platform/manage-deployments/upgrade-nodes/) for the recommended workflow.
+
+When following the above documented guidance, if you must perform an OS reboot on a node, be sure to drain or remove it from the cluster first. The recommended approach is to remove and then re-add the node.
+
+- Option 1: Blacklist the node Before Reboot (Drain): Blacklist the node, wait for draining to complete, reboot, and then remove the blacklist.
+- Option 2: (Recommended) Remove and add node: Remove the node from the cluster, patch or rebuild its OS, and then add the node back to the universe.
+When using either approach, work on one node at a time and wait for the cluster to return to full replication of tablets based on their RF before moving to the next node.
+
+##### Unplanned node outage (Failure scenario)
+
+In the event of an unplanned outage, assume the node's data is lost and proceed with a node replacement workflow (that is, using the Replace node universe action in YBA). If the node does come back on its own and starts to rejoin, monitor it closely.
+
+##### Pause universe or full-cluster shutdown
+
+This operation is not supported or safe with ephemeral disks. Stopping all nodes means every node's local disk is wiped, resulting in total data loss. Never shut down all nodes simultaneously. YBA does not perform this action for universes created using the on-premises infrastructure provider. But you must ensure your own automation scripts also avoid this.
+
+##### Software upgrades (YugabyteDB version upgrades)
+
+The [Upgrade Database Version](../../yugabyte-platform/manage-deployments/upgrade-software/) universe action in YBA does not reboot the machine or unmount disks. Therefore, this operation is safe. As per standard guidance during this operation, regardless of whether persistent or ephemeral disks are used, exercise caution in case a node fails during this operation as this may lead to a double fault scenario.
```

Comment:
```suggestion
In YugabyteDB Anywhere, the [Upgrade Database Version](../../yugabyte-platform/manage-deployments/upgrade-software/) option does not reboot the machine or unmount disks, and is therefore safe. As per standard guidance during this operation, regardless of whether persistent or ephemeral disks are used, exercise caution in case a node fails, as this may lead to a double fault scenario.
```